### PR TITLE
Styling unification.

### PR DESCRIPTION
**Pull request by tomislavhofman**
Related to #484

### CODE REVIEW

File: api/src/main/java/org/reactivestreams/Publisher.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
  * A {@link Publisher} is a provider of a potentially unbounded number of sequenced elements, publishing them according to
  * the demand received from its {@link Subscriber}(s).
  * <p>
- * A {@link Publisher} can serve multiple {@link Subscriber}s subscribed {@link #subscribe(Subscriber)} dynamically
+ * A {@link Publisher} can serve multiple {@link Subscriber}(s) subscribed {@link Publisher#subscribe(Subscriber)} dynamically
```
Comment (akarnokd) [inline api/src/main/java/org/reactivestreams/Publisher.java:18]: `multiple` already indicates plural here so `s` is fine.

File: api/src/main/java/org/reactivestreams/Processor.java [hunk 2]
```diff
@@ -12,11 +12,11 @@
...
 
 /**
- * A Processor represents a processing stage—which is both a {@link Subscriber}
+ * A {@link Processor} represents a processing stage—which is both a {@link Subscriber}
  * and a {@link Publisher} and obeys the contracts of both.
  *
- * @param <T> the type of element signaled to the {@link Subscriber}
- * @param <R> the type of element signaled by the {@link Publisher}
+ * @param <T> the type of element signaled to the {@link Subscriber}.
+ * @param <R> the type of element signaled by the {@link Publisher}.
```
Comment (akarnokd) [inline api/src/main/java/org/reactivestreams/Processor.java:19]: Such parameter descriptions are not full sentences themselves so I don't think `.` is needed here and elsewhere.

File: api/src/main/java/org/reactivestreams/Publisher.java (hunk 1 above)
Comment (tomislavhofman) [reply api/src/main/java/org/reactivestreams/Publisher.java:18]: Oops missed that, good catch.

File: api/src/main/java/org/reactivestreams/Processor.java (hunk 2 above)
Comment (tomislavhofman) [reply api/src/main/java/org/reactivestreams/Processor.java:19]: Alright, I'll remove those!

**Comment by viktorklang** [comment]
Hi @tomislavhofman,

Thanks for helping out with the documentation, much appreciated!

Would you also make sure you're added to the COPYRIGHTWAIVERS file?

√

**Comment by tomislavhofman** [comment]
Done it under first commit,
apologies for not making it clear!

**Comment by viktorklang** [comment]
@tomislavhofman No problem! (I didn't see it in the file list first for some reason)

**Comment by viktorklang** [comment]
Thanks @tomislavhofman!

**Comment by tomislavhofman** [comment]
Glad to contribute!

**Pull request merged**